frontend/src/components/regions/UserDashboard/UserDetailedStats.js

```diff
@@ -37,9 +37,17 @@ export default ({ detailedStats }) => {
           </TableCell>
         </TableRow>
         <TableRow>
-          <TableCell>Cards in complete column as assignee</TableCell>
```

Comment:
You aren't removing on what you already have, but adding for Status `Blocked`  and  `Ready`